app/models/user.rb

```diff
@@ -2,4 +2,5 @@ class User < ApplicationRecord
   include Clearance::User
 
   has_many :poems, dependent: :destroy
```

Comment:
For documentation: This really points to the fact that the current logic creates duplication around the user. While a poem might exist without a submission, it might make sense to disallow a submission with no poems, in which case I can make the submission's user come from it's poems (any one of them at this point).